tests/Silex/Tests/Provider/SpoolStub.php

```diff
@@ -36,7 +36,7 @@ public function isStarted()
 
     public function queueMessage(\Swift_Mime_Message $message)
     {
-        $this->messages[] = $message;
+        $this->messages[] = clone $message;
```

Comment:
Should probably be part of another PR as it does not seems related and not tested.